comp_conf/COMMON/build_scripts/build_UFS_UTILS.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+set -eu
+
+if [[ $(uname -s) == Darwin ]]; then
+  readonly MYDIR=$(cd "$(dirname "$(greadlink -f -n "${BASH_SOURCE[0]}" )" )" && pwd -P)
+else
+  readonly MYDIR=$(cd "$(dirname "$(readlink -f -n "${BASH_SOURCE[0]}" )" )" && pwd -P)
+fi
+
+SRW_APP_DIR="${MYDIR}/../../.."
+COMP_DIR="${SRW_APP_DIR}/comp_conf"
+SRC_DIR="${SRW_APP_DIR}/src/UFS_UTILS"
+BUILD_DIR="${SRW_APP_DIR}/build/UFS_UTILS"
+BIN_DIR="${SRW_APP_DIR}/bin/"
+
+# Detect MACHINE
+source ${COMP_DIR}/detect_machine.sh
+
+###########################################################
+## User specific parameters                              ##
+###########################################################
+##
+COMPILER="${COMPILER:-intel}"
+##
+###########################################################
+
+echo "MACHINE:" ${MACHINE}
+echo "COMPILER:" ${COMPILER}
+
+# file name suffix
+if [[ "${MACHINE}" == "hera" || "${MACHINE}" == "jet" || 
+    "${MACHINE}" == "orion" || "${MACHINE}" == "wcoss_dell_p3" ]]; then
+  FN_SFX=".lua"
+else
+  FN_SFX=""
+fi
+
+MODULE_FN="build.${MACHINE}.${COMPILER}${FN_SFX}"
+
+echo "Moduel file name:" ${MODULE_FN}
+
+# Load modules
+module purge
+module use ${SRC_DIR}/modulefiles
+module load ${MODULE_FN}
+module list
+
+# Copy module file to env
+cp "${SRC_DIR}/modulefiles/${MODULE_FN}" "${COMP_DIR}/env/modulefile.UFS_UTILS${FN_SFX}"
```

Comment:
We've had this discussion before, and while I see the utility of having a single env file per platform housed within the umbrella SRW App repo, my personal feeling is that it puts unnecessary work on the SRW App managers (us) to keep it working and current.  The other authoritative UFS repo managers keep these env files up to date and tested on all platforms already, why not take advantage of that and just copy them over at build/run time?  Then our modules are always in sync with whatever is being used for that hash of the repo in question.   With the variety of HPC platforms that exist, and the growing number of required modules, I'd just assume let the other code managers keep things working, and then we just leverage that work for the SRW App.  If there are separate build scripts for each component, then a component-specific "clean" shouldn't be a problem, right?  I also like the idea of being able to build one component separately within the App if I choose, instead of all together, as I've wanted that functionality when I only needed to rebuild ufs-weather-model, for example.